.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: trusty
+dist: xenial
```

Comment:
This is just the name of Ubuntu 16.10? Or we have other improvements here?

Reply:
It's a newer version and we could benefit for a improved performance: https://docs.travis-ci.com/user/reference/xenial/#differences-from-the-trusty-images